Automattic/wp-calypso
# Connections: Allow keyring connections to be shared by site users.

#### Changes proposed in this Pull Request

This is an experiment in allowing keyring connections to be shared across all users on a site, just like publicize connections are. 

#### Testing instructions

* Apply D35635-code to your sandbox, and sandbox `public-api.wordpress.com`.
* Create a new Eventbrite connection.
* Check the "Connection available to all..." checkbox.
